jwt.go

```diff
@@ -0,0 +1,25 @@
+package irma
+
+import (
+	"crypto/rsa"
+
+	"github.com/dgrijalva/jwt-go"
+)
+
+func CreateDisclosureJwt(attributes AttributeDisjunctionList, serverName, humanName string, sk *rsa.PrivateKey) (string, error) {
+	claims := ServiceProviderJwt{
+		ServerJwt: ServerJwt{
+			ServerName: serverName,
+			Type:       humanName,
```

Comment:
* `Type` should (in the case of disclosures) be the fixed string `verification_request`, see `NewServiceProviderJwt()`
* You can actually reuse `NewServiceProviderJwt()` to build the JWT before signing it
* We should have similar functions for signing and issuance (reusing `NewSignatureRequestorJwt()` and `NewIdentityProviderJwt()`

As to the `serverName` and `humanName`, that has now become a little messy:
* The variable `humanName` should be assigned to the field `ServerName`, whose name is inappropriate (I think `JwtIssuer` would be better
* The variable `serverName` should go into the JWT `kid` header.

Reply:
Ah, somehow I managed to miss `NewServiceProviderJwt()`. I've now reused this function.

> We should have similar functions for signing and issuance [...]

Yes, I'll add them later. I would prefer getting this one right first.

> The variable serverName should go into the JWT kid header.

Does this work? At least the Java implementation appears to use `iss`, not `kid` (verified by decoding the JWT for BIG name/birthdate disclosure).

Reply:
Originally we indeed used only `iss`; this was (1) shown to the user in the IRMA app and (2) the API server used this to lookup the verifier's public key with which to verify its JWTs. At some point we split these into two, so now:
* The API server uses the `kid` aka `serverName` field in the header for the public key lookup
* `iss` aka `humanName` is what is shown to the user in the IRMA app.

For example, here is the header from a decoded IRMATube JWT obtained by clicking on a movie:
```json
{
  "typ": "JWT",
  "alg": "RS256",
  "kid": "irmatube"
}
```
Body:
```json
{
  "iat": 1529269092,
  "iss": "IRMATube",
  "sub": "verification_request",
  "sprequest": { "truncated" }
}
```
These are made by [this PHP library](https://github.com/privacybydesign/irma-requestor/blob/master/src/Requestor.php).